### ISSUE TITLE
Update for Cactus for Mac?

### ISSUE THREAD

**Issue by mmasnjak** [koenbok/Cactus#153]
The mac client is super easy to onboard new designers and would be awesome if the mac client got an update. :)

**Comment by krallin** [comment]
@koenbok

**Comment by koenbok** [comment]
Ha I wish I had time for this :-)

But maybe I'll find some. We still use the app everyday here too.

**Comment by federojoavellaneda** [comment]
oh yeah, the high me ask you this incredible application go ahead!

**Comment by thomasoffinga** [comment]
@koenbok I definitely think the time is ripe for an update. If you need any testers on El Capitan, let me know! :)

**Comment by mattlovaglio** [comment]
I'd love to see an update as well. I totally respect that the team is working hard on Framer Studio. But I think Cactus For Mac is relevant enough that it get some new attention. +1

**Comment by koenbok** [comment]
At some point I will, but I really have to prioritize my startup.

Maybe if some people with cocoa desktop experience is interested, I can open source it.

**Comment by thomasoffinga** [comment]
@koenbok Yeah, you should definitely consider open sourcing it. While I'm not that well versed in Cocoa, I'd love to help improve the documentation. It's a bit outdated here and there.

**Comment by mfferreira** [comment]
+1

**Comment by ChristianBagley** [comment]
@koenbok +1 on open sourcing it for a mac update. I'd be very interested in getting in and helping on that project.

**Comment by cparnot** [comment]
@koenbok I use Cactus a lot for the findingsapp.com website. I am a Cocoa developer. +10000000 on open-sourcing!

**Comment by macfanatic** [comment]
I use Cactus to run a few sites and would be interesting in helping out for the Mac app.  If not then sharing that it won't be maintained would be a great notice to newcomers and direct them to use the CLI only.

**Comment by nathan-artist** [comment]
I agree that open sourcing the Cactus Mac App could be a good idea.
